changelog/unreleased/change-peerdependencies

```diff
@@ -0,0 +1,9 @@
+Change: Use peerDependencies instead of dependencies
+
+In the past, we used dependencies in package.json which lead to a big bundle size and increased the possibility 
+that the same package with 2 different versions is part of the final bundle of the consuming application.
+
+From now on, dependencies that are required to use the SDK are added to the peerDependencies section in package.json. 
```

Comment:
It's common practice to mention the peer dependencies in the install commands in the `Readme.md` as well. Since we list install commands there I'd appreciate to see them added there for us as well.

Reply:
We should do the same in the ODS, then? But good point, working on it

Reply:
yep, that would be a nice improvement 👍

Reply:
e.g. https://www.npmjs.com/package/vue-datetime#bundler-webpack-rollup